src/services/time-entries-api.tsx

```diff
@@ -19,28 +22,34 @@ export async function getTimeEntries(): Promise<Types.TimeEntryProps[]> {
 }
 
 export async function addTimeEntry(newTimeEntry: Types.TimeEntryProps) {
-  const response = await fetch("http://localhost:3004/time-entries", {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json",
+  const response = await fetch(
+    "https://my-json-server.typicode.com/humanoidsbv/team-awesome-fabiano/time-entries",
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(newTimeEntry),
     },
-    body: JSON.stringify(newTimeEntry),
-  });
+  );
   return response.json();
 }
 
 export async function removeTimeEntry(id: number) {
-  const response = await fetch("http://localhost:3004/time-entries/" + id, {
-    method: "DELETE",
-    headers: {
-      "Content-Type": "application/json",
+  const response = await fetch(
+    "https://my-json-server.typicode.com/humanoidsbv/team-awesome-fabiano/time-entries/" + id,
```

Comment:
I would write this as `https://my-json-server.typicode.com/humanoidsbv/team-awesome-fabiano/time-entries/${id}`.